### PR TITLE
CCD Apps Not Working Develop-to-Develop on Preview

### PR DESCRIPTION
**Pull request by sumansreenivasan**
### JIRA link ###

https://tools.hmcts.net/jira/browse/RDM-8995

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,7 +19,6 @@
     "test:coverage": "NODE_ENV=mocha LOG_LEVEL=OFF nyc mocha --compilers ts:ts-node/register -r tsconfig-paths/register $(find src/test \\( -name '*.ts' -or -name '*.js' \\) ! -path '*a11y*')",
     "test:smoke": "echo SMOKE TESTS NOT CONFIGURED",
     "test:functional": "echo 'TODO - FUNCTIONAL TEST'",
-    "test:audit": "./bin/run-yarn-audit.sh -f yarn-audit-known-issues",
```
Comment (smathangi) [inline package.json:22]: what's the reason for this change?

Comment (smathangi) [reply package.json:22]: aha..just saw related jenkins file change . Ignore above comment.

**Review by MSancaktutar — APPROVED** [review]
LGTM
